pkg/kafka/producer/config.go

```diff
@@ -32,6 +34,8 @@ type Configuration struct {
 	Compression      sarama.CompressionCodec
 	CompressionLevel int
 	ProtocolVersion  string
+	LingerMS         int
```

Comment:
we prefer typed values:

```suggestion
	BatchLinger         time.Duration
```

Reply:
Changed to the suggested name and type in the latest commit.